samples/modules/cmsis_dsp/moving_average/sample.yaml

```diff
@@ -0,0 +1,48 @@
+sample:
+  description: Use CMSIS DSP to calculate moving average
+  name: CMSIS DSP Moving Average
+tests:
+  sample.modules.cmsis_dsp.moving_average:
+    tags:
+      - samples
+    integration_platforms:
+      - qemu_cortex_m0
```

Comment:
Using `mps2_an521_remote` here would be a nice opportunity to show MVE in action :).